TrayTip/__init__.py

```diff
@@ -62,7 +62,7 @@ def OnNotify(self, hwnd, msg, wparam, lparam):
         if lparam == 0x0404 or lparam == 0x0405:
             win32gui.DestroyWindow(hwnd)
 
-class showTip(eg.ActionBase):
+class ShowTip(eg.ActionBase):
```

Comment:
Wouldn't you also need to change the call to self.addAction() on line 26 (in the main TrayTip class)?